src/components/AboutUs/AboutMedicare.jsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+import AppButton from "@/UI/AppButton";
+import ComponentHeader from "@/UI/ComponentHeader";
+import Image from "next/image";
+import { useState } from "react";
+
+export default function AboutMedicare() {
+  const [isVideoOpen, setIsVideoOpen] = useState(false);
+
+  const toggleVideoPopup = () => {
+    setIsVideoOpen(!isVideoOpen);
+  };
+
+  return (
+    <>
+      <div className="max-w-7xl mx-auto flex flex-col lg:flex-row items-center gap-10">
+        {/* Left Section */}
+        <div className="relative w-full md:w-1/2">
+          <div className="relative">
+            <Image
+              width={400}
+              height={500}
+              src="/assets/about_mdc.png"
+              alt="Doctor"
+              className="rounded-lg w-full"
+            />
+            <button
+              onClick={toggleVideoPopup}
+              className="absolute inset-0 m-auto flex items-center justify-center w-16 h-16 bg-blue-600 rounded-full shadow-md hover:bg-blue-700 focus:outline-none transition-all duration-200"
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                fill="white"
+                viewBox="0 0 24 24"
+                strokeWidth="1.5"
+                stroke="currentColor"
+                className="w-10 h-10"
+              >
+                <path
+                  strokeLinecap="round"
+                  strokeLinejoin="round"
+                  d="M6.75 12l7.5-4.5v9l-7.5-4.5z"
+                />
```

Comment:
Keep the svg file on customSvg folder on the code base and import it from there.